service/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
     liquibaseRuntime 'org.liquibase:liquibase-core:4.8.0'
     liquibaseRuntime 'info.picocli:picocli:4.6.1'
     liquibaseRuntime 'org.postgresql:postgresql:42.4.1'
-    liquibaseRuntime 'ch.qos.logback:logback-classic:1.2.9'
+    liquibaseRuntime 'ch.qos.logback:logback-classic'
```

Comment:
Why do we want to completely remove the pinned version, vs moving to a more recent version?